spec/services/post_alerter_spec.rb

```diff
@@ -2004,4 +2004,33 @@ def create_post_with_incoming
       expect(liked_notification.data_hash[:custom_key]).to eq(custom_data)
     end
   end
+
+  it "does not create notifications for PMs if not invited" do
+    SiteSetting.pm_tags_allowed_for_groups = "#{Group::AUTO_GROUPS[:everyone]}"
+    tag_1 = Fabricate(:tag)
```

Comment:
How about we use a more explicit name?
```suggestion
    watched_tag = Fabricate(:tag)
```